src/Platforms/AbstractPlatform.php

```diff
@@ -98,6 +99,25 @@ abstract class AbstractPlatform
      */
     protected ?KeywordList $_keywords = null;
 
+    /**
+     * Defines how the platform folds the case of unquoted identifiers.
+     */
+    private ?UnquotedIdentifierFolding $unquotedIdentifierFolding = null;
+
+    protected function __construct(?UnquotedIdentifierFolding $unquotedIdentifierFolding = null)
```

Comment:
Adding protected constructor for AbstractPlatform will break 3rd party extenders of this abstract class (who don't have explicit public constructor).

Should this change (make constructor protected) be done in major release, not minor?

Reply:
Good catch. I didn't think of this. I will make it public for now.